docs/docfx.json

```diff
@@ -1,4 +1,26 @@
 {
+  "metadata": [
+    {
+      "src": [
+        {
+	  "src": "../",
+          "files": [
+	    "src/OoLunar.DSharpPlus.CommandAll.csproj"
```

Comment:
The file in general isn't formatted

Reply:
i had to do all work in nano, can you perhaps format it?

Reply:
Yeah I can do it.